django_multitenant/tests/gis_models.py

```diff
@@ -0,0 +1,22 @@
+import uuid
```

Comment:
We need more tests to test all the cases
I think there should be a test for PostGIS models which will be correspond to Regular models. 
We are covering lots of cases  there